pymsis/msis.py

```diff
@@ -378,14 +378,15 @@ def create_input(
     """
     # Turn everything into arrays
     dates_arr: npt.NDArray[np.datetime64] = np.atleast_1d(
-        np.array(dates, dtype=np.datetime64)
+        np.asarray(dates, dtype=np.datetime64)
     )
```

Comment:
Would it be better to invert this and remove the asarray?

`np.atleast_1d(dates).astype(np.datetime64)`